src/main/java/teammates/ui/controller/AdminSearchPageAction.java

```diff
@@ -347,7 +354,11 @@ private AdminSearchPageData extractDataFromFeedbackSeesion(FeedbackSessionAttrib
                 data.studentPublishedFeedbackSessionLinksMap.get(student.getIdentificationString()).add(viewResultUrl);
             }
 
-            data.feedbackSeesionLinkToNameMap.put(viewResultUrl, fsa.getFeedbackSessionName() + " (Published)");
+            data.feedbackSeesionLinkToNameMap.put(viewResultUrl, fsa.getFeedbackSessionName()
+                    + " (Published)"
+                    + "(" + fsa.getSessionStartTime()
+                    + " - " + fsa.getSessionEndTime()
+                    + ")");
```

Comment:
Perhaps use `[` and `]` around the period because `()` already used once (for `(Published)`)?